pkgs/development/compilers/open-watcom/v2.nix

```diff
@@ -12,14 +12,14 @@
 
 stdenv.mkDerivation rec {
   pname = "open-watcom-v2";
```

Comment:
```suggestion
  pname = "open-watcom-v2-unwrapped";
```

Reply:
This is missing the corresponding changes in `bin.nix` and `wrapper.nix` for consistency and the derivation names to make any sense, otherwise the wrapped ones will inherit the `-unwrapped` suffix.

Reply:
Then this requires more changes but overwriting name with something that is not constructed from pname+version is a bit ugly.

Reply:
Should I put the project name into a passthru attribute then so the wrapper can use it? Please suggest something better.

I know this way of doing it is ugly but FWIW this was basically taken from the FF derivation, where the unwrapped versions manually construct tge name like that and the wrapper takes pname & version:
https://github.com/NixOS/nixpkgs/blob/6a894dc68d3fc16de7f52b235b2931a98c8e0b9b/pkgs/applications/networking/browsers/firefox/common.nix#L205

Reply:
I think that might be missing pname which is also not great. Using passthru would be alright, I think. I don't know a better way right now other than hardcoding.